### PR TITLE
11268

### PR DESCRIPTION
**Pull request by faustino-garcia**
Tech debt: "Chefname" column of chefs table changed to "name"

### CODE REVIEW

File: app/views/chefs/_form.html.erb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 <div class="row">
   <%= form_for(@chef, :html => {role: "form"}) do |f| %>
     <div class="col mb-3">
-      <%= f.label :chefname, "Choose Chefname", class: "form-label" %>
+      <%= f.label :name, "Choose name", class: "form-label" %>
```
Comment (coderdave) [inline app/views/chefs/_form.html.erb:6]: Changing the label was not in the AC but I think it makes sense. Hopefully the PO approves it!

**Review by coderdave — APPROVED** [review]
I also switched to the branch and smoke tested a few pages and it works. All the tests pass too. Good job!

**Review by AlexRS90 — APPROVED** [review]
Hi team, 
Good job refactoring the name of the column, everything is working as expected!

Cheers!

**Review by vicmaburrito — APPROVED** [review]
Great Job so far, looks great!

**Review by ArmandoD97 — APPROVED** [review]

**Review by juantintos — APPROVED** [review]
Good Job 👍

**Pull request merged**